library/MedEx/MedEx_background.php

```diff
@@ -0,0 +1,80 @@
+<?php
+/**
```

Comment:
lol,
turns out this script is a function.
So this just pertains to the script above.